com.unity.render-pipelines.high-definition/Editor/Material/UIBlocks/DetailInputsUIBlock.cs

```diff
@@ -76,13 +75,12 @@ internal class Styles
         /// <param name="layerIndex">Current layer index to display. 0 if it's not a layered shader</param>
         /// <param name="features">Features of the block.</param>
         /// <param name="dotColor">Subheader dot color. See Layered Lit UI subheader for more info.</param>
-        public DetailInputsUIBlock(ExpandableBit expandableBit, int layerCount = 1, int layerIndex = 0, Features features = Features.All, Color dotColor = default(Color))
+        public DetailInputsUIBlock(ExpandableBit expandableBit, int layerCount = 1, int layerIndex = 0, Features features = Features.All)
```

Comment:
since it is public we should probably leave the previous constructor as is (with Color dotcolor), mark it as obsolete and call the new constructor without the Color dotColor, what do you think?

Reply:
Perfect, I will do that.

Reply:
@jenniferd-unity , I've tried to do it obsolete but as this is an argument with default parameter I am getting.

```
![image](https://user-images.githubusercontent.com/76204843/109274320-a7365680-7813-11eb-8cab-0964169d7a4b.png)
```
What do you think is the best here to do?, leave the argument and do not use it, or just remove it?